lib/matplotlib/rcsetup.py

```diff
@@ -718,6 +719,54 @@ def visit_Attribute(self, node):
         self.generic_visit(node)
 
 
+def validate_legend_loc(loc):
+    """
+    Confirm that loc is a type which rc.Params["legend.loc"] supports.
+
+    .. versionchanged:: 3.7
```

Comment:
I think this should be `versionadded` rather than `versionchanged`, since it is a new function.  Should probably also update the version to 3.8 since this PR is in that milestone.